source/gui/logViewer.py

```diff
@@ -80,6 +80,7 @@ def onSaveAsCommand(self, evt):
 		try:
 			# codecs.open() forces binary mode, which is bad under Windows because line endings won't be converted to crlf automatically.
 			# Therefore, do the encoding manually.
+			# #9038 (Py3 review required): although eligible for "with open" function conversion, encoding issue must be settled first.

```

Comment:
Per the documentation for open in Python3, providing no encoding causes it to default to locale.getpreferredencoding(False). On Windows (at least my machine) locale.getpreferredencoding(False) is cp1252 which only supports the Latin alphabet. I think we should provide encoding='utf-8' to the call to open, and then we can remove the .encode('utf-8') from the string when calling write.